guide/release-notes/main.adoc

```diff
@@ -14,6 +14,8 @@ IMPORTANT: Changes in code lists are also reflected by changes in the validation
 
 :leveloffset: +1
 
+include::v3.0.9-hotfix.adoc[]
```

Comment:
Shouldn't this be v3.0.8-hotfix, as the last release was called https://github.com/OpenPEPPOL/peppol-bis-invoice-3/releases/tag/3.0.8

??